web-app/src/app/websocket.service.ts

```diff
@@ -15,7 +15,7 @@ export class WebSocketService {
   lapMultiplex$: Observable<any>;
 
   constructor() {
-    this.socket$ = webSocket('ws://localhost:4000');
+    this.socket$ = webSocket('ws://138.197.206.227:4000');
```

Comment:
Was this just for the race? Does it need to be reverted?

Reply:
This was so it would deploy to the server. That should be reverted so it doesn't go to master